### PR TITLE
Cdh workaround

### PR DESCRIPTION
**Pull request by mvfast**
@fjy 
added workaround solution doc for cdh jackson conflict issue.

### CODE REVIEW

**Comment by fjy** [comment]
@mvfast Awesome! Thanks for the contrib! Do you mind filling out our CLA (druid.io/community/cla.html)? Can you also squash the commits and after that I'll merge

**Comment by drcrallen** [comment]
Cool, is it possible to have these exclusions as a maven profile?

File: docs/content/operations/use_sbt_to_build_fat_jar.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+libraryDependencies ++= Seq(
```
Comment (xvrl) [inline docs/content/operations/use_sbt_to_build_fat_jar.md]: would you mind adding the yaml front-matter and format the sbt code in a code block? That would make it friendlier to use.

**Comment by mvfast** [comment]
@drcrallen I think you can exclude the dependencies in pom file too but I haven't tested that. For me, the sbt solution seems to be easier.

**Comment by fjy** [comment]
@mvfast Instructions for squashing and rebasing:
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by fjy** [comment]
@mvfast did you get a chance to fill out the CLA? druid.io/community/cla.html

**Comment by mvfast** [comment]
@fjy Yeah. I need to ask my manager which CLA to sign(company or individual). I'll reply to you today. Thanks!

**Comment by mvfast** [comment]
@fjy Hi Fangjin, Just let you know I've filled out the CLA.

**Comment by fjy** [comment]
@mvfast Awesome! thanks :)

**Comment by fjy** [comment]
@mvfast BTW, if you want your contribution to count publicly, you may want to add your email to your github profile:
https://groups.google.com/forum/#!searchin/druid-development/credit/druid-development/1Zs3Vd9LHCQ/cadqT73UeVcJ

**Pull request merged**